erigon-lib/commitment/hex_patricia_hashed.go

```diff
@@ -1543,12 +2030,12 @@ func (hph *HexPatriciaHashed) Process(ctx context.Context, updates *Updates, log
 		if stateUpdate == nil {
 			// Update the cell
 			if len(plainKey) == hph.accountKeyLen {
-				update, err = hph.ctx.Account(plainKey)
+				update, err = hph.Ctx.Account(plainKey)
```

Comment:
why need to expose Ctx?

Reply:
I did it for easier debugging, I guess I can change this back to what it was.